client/my-sites/checkout/checkout-thank-you/marketplace/marketplace-thank-you.tsx

```diff
@@ -151,6 +151,8 @@ const MarketplaceThankYou = () => {
 				thankYouImage={ thankYouImage }
 				/* TODO: Change thank you message to be dynamic according to product */
 				thankYouTitle={ translate( 'Yoast SEO Premium is installed' ) }
+				headerBackgroundColor={ '#f5f7f7' }
+				headerTextColor={ 'black' }
```

Comment:
I thought we wanted all these components to use the new colours. The older design had this light grey background, but I think all of the new designs should use the new colour scheme.

Either way, I think the team working on this is going to be updating this component soon, so I'd rather default to the new screen for now, and they can decide otherwise when they get to work on this.

Reply:
Oh sure, I thought Yoast page had to remain the same as before.